### ISSUE TITLE
Is there any way to save bone mapping list?

### ISSUE THREAD

**Issue by RavenLeeANU** [Rokoko/rokoko-studio-live-blender#119]
Hi Rokoko, Thanks for your great works for supporting retargeting process in blender. I use this plugin for binding multiple animation clips, which have same skeleton. I wonder is there any way to save bone mapping list I just set down so that I can reuse it every time I import the animation with the same skeleton. Appreciate!

**Comment by SnowdenWintermute** [comment]
Did you ever figure out how to do this? I need to retarget several animations from multiple skeletons with the exact same bone structure.

**Comment by z-moose** [comment]
Yes this would be amazing and save so much time. I'm importing multiple animations from Mixamo and have to manually do the bone list each time I import a Mixamo FBX to retarget it to the same Rigify Human Meta-Rig.
Using Blender 4.4.3 and Rokoko Studio Live v1.4.3
Thanks for the amazing add-on!
